src/replit/web/app.py

```diff
@@ -73,6 +73,7 @@ def debug(
     **kwargs: Any
 ) -> None:
     """Run the app in debug mode.
+
```

Comment:
Intentional?

Reply:
Yep! The lack of this newline was causing a warning in the doc generation.